fix: number card layout

fadilsiddique opened the pull request:
Changed Number Card CSS property from justify-content: flex-start to justify-content: space-between.
![Admin-Portal](https://github.com/frappe/frappe/assets/48912196/ffea9d9c-8a8b-475d-889a-abf626305827)

![Screenshot from 2023-11-20 07-11-20](https://github.com/frappe/frappe/assets/48912196/d2c153a6-8093-460c-976b-f82069413c57)

fadilsid committed:
fix: numbercard layout

shariquerik commented:
@fadilsiddique The issue occurred after this https://github.com/frappe/frappe/commit/a46739688e9d9b2b3da99fbae803325571db35c9#diff-877950bba0044107908a4275923ea0a1acbb080036188485e1f1a46ddb7ab2e0

cc: @maharshivpatel 

Lets reset all `justify-between` in above mentioned commit and only keep `flex-start` for .shortcut-widget-box

fadilsiddique commented:
@fadilsiddique You can make above changes that I suggested in this PR

fadilsid committed:
Merge branch 'develop' into numbercard_fix

fadilsiddique commented:
> .shortcut-widget-box

There is no class .shortcut-widget-box in above PR.

fadilsiddique commented:
@maharshivpatel  can you review this PR?

maharshivpatel committed:
fix: shortcut-widget-box should have flex-start

As we changed align-items to space-between for widget head,
we need to override .shortcut-widget-box to flex-start

maharshivpatel committed:
Merge branch 'develop' into numbercard_fix
